lib/virt.py

```diff
@@ -819,6 +827,51 @@ def ssh_keygen(path):
     subprocess.run(['ssh-keygen', '-N', '', '-f', path], stdout=DEVNULL, check=True)
 
 
+def translate_ssg_kickstart(profile):
+    """
+    Parse (and tweak) a kickstart shipped with the upstream content
+    into class Kickstart instance.
+    """
+    ks_text = ''
+    with open(util.get_kickstart(profile)) as f:
+        for line in f:
+            line = line.rstrip('\n')
+
+            # use our own password
+            if re.match('^rootpw ', line):
+                line = f'rootpw {GUEST_LOGIN_PASS}'
+
+            # don't hardcode interface name
+            if re.match('^network ', line):
```

Comment:
As you are cycling over lines, there will be single match every time. Thus, to make it obvious, use `elif` statement here and further.

Reply:
I guess I saw the matches as more independent (easier to add/remove when there's no `elif`), but I can make that optimization, sure.